Baremetal-Blinky/README.md

```diff
@@ -0,0 +1,32 @@
+## Baremetal Blinky example using Mbed OS
+
+This guide reviews the steps required to build and run a baremetal Blinky application on Ti's CC3220SF LaunchXL platform without using DigitalOut class and timers.
+
+Please install [Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
+
+This example retrieves code from https://github.com/ARMmbed/mbed-os-ti-port. It's a private repo, please contact support@mbed.com to request access.
```

Comment:
Any reason to keep this repository private? Seems counter intuitive for an example!

Reply:
Think it might have to do with where this doc is being referenced to begin with? 

@AnotherButler Where is this example living in the docs anywho?